DEVELOPER_GUIDE.md

```diff
@@ -0,0 +1,87 @@
+- [OpenSearch SDK Developer Guide](#opensearch-sdk-developer-guide)
+  - [Introduction](#introduction)
+  - [Getting Started](#getting-started)
+    - [Git Clone IndependentPlugin Repo](#git-clone-independentplugin-repo)
+	- [Run Independent Plugin](#run-independent-plugin)
+	- [Run Tests](#run-tests)
+	- [Send Message using Telnet](#send-message-using-telnet)
+
+# OpenSearch SDK Developer Guide
+
```

Comment:
Can you move the index here? Also, you can remove `OpenSearch SDK Developer Guide` from the index.